### PR TITLE
test: convert //test/integration:ratelimit_integration_test to proto.

### PR DESCRIPTION
**Pull request by htuch**
Signed-off-by: Harvey Tuch <htuch@google.com>

### CODE REVIEW

**Comment by htuch** [comment]
@alyssawilk @mattklein123

File: test/integration/legacy_json_integration_test.cc [hunk 1]
```diff
@@ -69,10 +69,6 @@ TEST_P(LegacyJsonIntegrationTest, TestServerGrpcJsonTranscoder) {
   createTestServer("test/config/integration/server_grpc_json_transcoder.json", {"http"});
 }
 
-TEST_P(LegacyJsonIntegrationTest, TestServerRatelimit) {
```
Comment (mattklein123) [inline test/integration/legacy_json_integration_test.cc]: Any particular reason to remove from the legacy integration config tests?

Comment (htuch) [reply test/integration/legacy_json_integration_test.cc]: Just mopping up the JSON, we can keep it there if you prefer. I think we have sufficient coverage from the proto-based test.

Comment (mattklein123) [reply test/integration/legacy_json_integration_test.cc]: It's not a big deal. I still feel like we are losing some ease of inspection of what is actually being tested without being able to scan the full configs, but I will get over it.

**Pull request merged**